src/avt_vimba_camera.cpp

```diff
@@ -803,13 +809,17 @@ bool AvtVimbaCamera::createParamFromFeature(const FeaturePtr feature, std::strin
         feature->GetValue(initial_value);
         feature->GetRange(minimum_value, maximum_value);
         feature->GetIncrement(step);
+        
+        initial_value = std::clamp(initial_value, minimum_value, maximum_value);
```

Comment:
When I compile this on Foxy, I get the following error:
`error: ‘clamp’ is not a member of ‘std’`

Foxy uses C++14 by default, which doesn't seem to include `clamp`.

Reply:
Ah yes, this feature is from C++17. I will revert this to min max clamping.